karapace/schema_reader.py

```diff
@@ -248,56 +249,59 @@ def handle_messages(self) -> None:
                 try:
                     key = ujson.loads(msg.key.decode("utf8"))
                 except ValueError:
-                    self.log.exception("Invalid JSON in msg.key: %r, value: %r", msg.key, msg.value)
+                    self._log.exception("Invalid JSON in msg.key")
+                    self._log.log(TRACE, "Invalid JSON in msg.key: %r, value: %r", msg.key, msg.value)
```

Comment:
I'm not really fond of this non-standard `TRACE` level logging. I think we should just go with debug level logging for these, which is kind of standard, but not for production usage.

Reply:
Any other arguments? That was intentional as trace is more verbose than debug.

Reply:
Removed all TRACE calls.